scripts/apps/search/directives/SearchParameters.ts

```diff
@@ -78,6 +78,14 @@ export function SearchParameters($location, asset, tags, metadata, common, desks
                 scope.search_config = getSearchConfig();
                 scope.lookupCvs = {};
                 scope.params = params.params ? JSON.parse(params.params) : {};
+
+                if (scope.query && !Object.keys(scope.meta).length) {
```

Comment:
Make it more readable. What are you trying to match with this regex? add an example.

There are too many things done in lines 85,86. I can't see where are the boundaries of the functions and which operations take place in which order since everything is that condensed. Use clearer variable names - `query` instead of `t`, `key` and `value` instead of `k`,`v`.